inductiva/tasks/methods.py

```diff
@@ -1,15 +1,48 @@
 """Methods to interact with the tasks submitted to the API."""
+from collections import defaultdict
 import json
-from typing import Dict, List, Optional, Union, Sequence
-
-import pandas as pd
+from typing import Any, Dict, List, Mapping, Optional, Union
 
 import inductiva
 from inductiva import api
 from inductiva.client import ApiClient, ApiException
 from inductiva.client.apis.tags.tasks_api import TasksApi
 from inductiva.client import models
-from inductiva.utils import format_utils
+
+
+def to_dict(list_of_tasks: List) -> Mapping[str, List[Any]]:
```

Comment:
type hint can be enhanced;
missing docstring.

```suggestion
def to_dict(list_of_tasks: Iterable[Task]) -> Mapping[str, List[Any]]:
```